automated/linux/imagenet-resnet50/imagenet-resnet50.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# shellcheck disable=SC1091
+
+. ../../lib/sh-test-lib
+OUTPUT="$(pwd)/output"
+
```

Comment:
If you only run this script you need to set these variables too

HOME_DIR='/home/debiand05'
MNT_DIR='/mnt'
MNT_EXISTS=true

if not wont it be messy?